tests/karate/src/test/java/be/utils/GoServer.java

```diff
@@ -20,13 +20,13 @@ public String[] getCmd() {
       return new String[]{
         "cmd",
         "/c",
-        "\"pop.exe server --pk J9fBzJV70Jk5c-i3277Uq4CmeL4t53WDfUghaK0HpeM= serve\""
+        "\"pop.exe server serve\""
       };
     } else {
       return new String[]{
         "bash",
         "-c",
-        "./pop server --pk J9fBzJV70Jk5c-i3277Uq4CmeL4t53WDfUghaK0HpeM= serve"
+        "./pop server serve"
```

Comment:
I have a question :
How is the organizer key given to the server without this argument ?

Reply:
Oh ok, that's something I didn't follow. Thanks for the clarification.